src/Sulu/Component/CustomUrl/Manager/RouteAlreadyExistsException.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\CustomUrl\Manager;
+
+use Sulu\Component\Rest\Exception\RestException;
+
+/**
+ * Thrown when route already exists.
+ */
+class RouteAlreadyExistsException extends RestException
```

Comment:
Isn't that basically telling the same as the `ResourceLocatorAlreadyExistsException`? The only instantiation creation of this class even is in combination with a `RouteDocument`.